dotnet/src/Extensions/PromptTemplate.Handlebars/HandlebarsPromptTemplateOptions.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using HandlebarsDotNet;
+using HandlebarsDotNet.Helpers.Enums;
+using HandlebarsDotNet.Helpers.Options;
+
+#pragma warning disable IDE0130 // Namespace does not match folder structure
+namespace Microsoft.SemanticKernel.PromptTemplate.Handlebars.Helpers;
+#pragma warning restore IDE0130
+
+/// <summary>
+/// Configuration for Handlebars helpers.
+/// </summary>
+public sealed class HandlebarsPromptTemplateOptions : HandlebarsHelpersOptions
+{
+    // TODO [@teresaqhoang]: Add Categories filter for KernelSystemHelpers (i.e., KernelHelperCategories)
+
+    /// <summary>
+    /// Default character to use for delimiting plugin name and function name in a Handlebars template.
+    /// </summary>
+    public string DefaultNameDelimiter { get; set; } = "-";
+
+    /// <summary>
+    /// Delegate for registering custom helpers.
+    /// </summary>
+    /// <param name="handlebarsInstance">The Handlebars instance.</param>
+    /// <param name="executionContext">Arguments maintained in the template execution context.</param>
+    public delegate void RegisterCustomHelpersCallback(IHandlebars handlebarsInstance, KernelArguments executionContext);
```

Comment:
Do we need a custom delegate, or could we just use `Action<IHandlebars, KernelArguments>`?

Reply:
Changed to `Action<IHandlebars, KernelArguments>`